tests/php/ORM/DataObjectSchemaTest.php

```diff
@@ -159,68 +159,94 @@ public function testTableForObjectField()
         );
     }
 
-    public function testFieldSpec()
+    public function provideFieldSpec()
```

Comment:
```suggestion
    public function provideFieldSpec(): array
```
Also move the provide method below the test method

Reply:
Done